docker-compose.yml

```
@@ -2,55 +2,65 @@ version: "2"
 services:
   tf2:
     image: team-brh/tf2:latest
-    dns: 
+    dns:
       - 172.28.0.2
-    networks: 
+    networks:
       - game-network
-    depends_on: 
+    depends_on:
       - dns
       - cache
-  
+
   css:
     image: team-brh/css:latest
-    dns: 
+    dns:
       - 172.28.0.2
-    networks: 
+    networks:
       - game-network
-    depends_on: 
+    depends_on:
       - dns
       - cache
     volumes:
       - fast_download:/fastdownload:ro
 
   pz:
     image: team-brh/pz:latest
-    dns: 
+    dns:
       - 172.28.0.2
-    networks: 
+    networks:
       - game-network
-    depends_on: 
+    depends_on:
       - dns
       - cache
     volumes:
       - pz_db:/home/steam/Zomboid/db
       - pz_save:/home/steam/Zomboid/Saves
 
+  l4d2:
+    image: team-brh/l4d2:latest

```

This won't pick up the DNS and cache servers without being on the same network:
```suggestion
    image: team-brh/l4d2:latest
    networks:
      - game-network    
```